woocommerce/woocommerce
# Shipping Settings: Add currency symbols

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Fixes https://github.com/woocommerce/woocommerce/issues/40737



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a Shipping Zone `/wp-admin/admin.php?page=wc-settings&tab=shipping`
2. Add a Free Shipping Method
3. See the cost input, compare to designs at TgDI6pJuBmnPyZ810B7uzX-fi
4. Try entering an invalid cost, see the validation fail. This is the default experience, I didn't change it.
5. See the price inputs for Flat Rate and Local Pickup options
6. Add a shipping class `/wp-admin/admin.php?page=wc-settings&tab=shipping&section=classes`, then go back and edit the Flat Rate method. There will be additional price inputs



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add currency symbol to Shipping Settings price inputs

#### Comment 

</details>
